<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DES-cbc</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>881</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>密钥:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="2" column="9">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="2" column="7">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string> 解密文件</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7" rowspan="4" colspan="3">
      <widget class="QPlainTextEdit" name="plainTextEdit_2">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" rowspan="4" colspan="3">
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>加密文件</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5" rowspan="2">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>&lt;- 开始解密</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5" rowspan="2">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>开始加密 -&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Hex</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>Text</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1" colspan="9">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="2" column="6" rowspan="5">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="3" rowspan="5">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
